setup.rst

```diff
@@ -341,6 +341,14 @@ Now you can install the build dependencies via ``apt``::
 If that package is not available for your system, try reducing the minor
 version until you find a package that is available.
 
+If you want to build all modules, install these packages in the versions you
+prefer, changing the version name as above::
```

Comment:
I'm also not a fan of the "versions you prefer..." bit, I would suggest to trim it back to just 
```suggestion
If you want to build all optional modules, install the following packages and
their dependencies::
```

Reply:
+1

Reply:
I have accepted this suggestion entirely, merging it with the deletion of the previous paragraph.